api/controllers/game/stalemate.js

```diff
@@ -6,12 +6,17 @@ module.exports = async function (req, res) {
   try {
     const { game: gameId, pNum, usr: userId } = req.session;
     // Track which turn each player most recently requested stalemate
-    let game = await Game.findOne({ id: gameId })
-      .populate('players', { sort: 'pNum' });
+    let game = await Game.findOne({ id: gameId }).populate('players', { sort: 'pNum' });
     let gameUpdates = {};
     const keyPrefix = 'turnStalemateWasRequestedByP';
     const playerStalemateKey = `${keyPrefix}${pNum}`;
     const playerStalemateVal = game.turn;
+
+    // Error if this player already requested stalemate this turn
+    if (game[playerStalemateKey] === game.turn) {
+      throw new Error('game.snackbar.stalemate.previousStalemateRejected');
+    }
```

Comment:
Adding this error to match new GameState API requirement as described above. You can no longer request a stalemate more than once per turn.